Makefile.calico-node

```diff
@@ -240,13 +249,14 @@ st: dist/calicoctl busybox.tar routereflector.tar calico-node.tar workload.tar r
 	           --rm -ti \
 	           -v /var/run/docker.sock:/var/run/docker.sock \
 	           -v $(SOURCE_DIR):/code \
+	           -v $(SOURCE_DIR)/tests:/tests \
 	           $(SYSTEMTEST_CONTAINER) \
-	           sh -c 'cp -ra tests/st/* /tests/st && cd / && nosetests $(ST_TO_RUN) -sv --nologcapture --with-timer $(ST_OPTIONS)'
+	           sh -c 'cd / && nosetests $(ST_TO_RUN) -sv --nologcapture --with-timer $(ST_OPTIONS)'
```

Comment:
Since we control this image, can we just change the WORKDIR in the dockerfile to avoid this `cd`